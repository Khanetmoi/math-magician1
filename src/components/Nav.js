import React from 'react';
import { Link } from 'react-router-dom';

const Nav = () => (
  <div className="flex">
    <h2>Math magicians</h2>
    <nav>
      <Link className="navLink" to="/">Home</Link>
      <Link className="navLink" to="/Calculator">Calculator</Link>
      <Link className="navLink" to="/Quotes">Quote</Link>
    </nav>
  </div>
);

export default Nav;
