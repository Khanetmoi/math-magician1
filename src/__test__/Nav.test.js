import React from "react";
import renderer from 'react-test-renderer';
import { HashRouter as Router } from "react-router-dom";
import Nav from "../components/Nav";
import '@testing-library/jest-dom';


describe('Test of Nav', () => {
    it('test if Header is rendered', () => {
        const headerTest = renderer
            .create(<Router><Nav /></Router>)
            .toJSON();
        expect(headerTest).toMatchSnapshot();
    });
});