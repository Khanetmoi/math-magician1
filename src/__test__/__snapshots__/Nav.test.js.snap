// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of  test if Header is rendered 1`] = `
<div
  className="flex"
>
  <h2>
    Math magicians
  </h2>
  <nav>
    <a
      className="navLink"
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="navLink"
      href="#/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="navLink"
      href="#/Quotes"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;

exports[`Test of Header test if Header is rendered 1`] = `
<div
  className="flex"
>
  <h2>
    Math magicians
  </h2>
  <nav>
    <a
      className="navLink"
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="navLink"
      href="#/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="navLink"
      href="#/Quotes"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;

exports[`Test of Nav test if Header is rendered 1`] = `
<div
  className="flex"
>
  <h2>
    Math magicians
  </h2>
  <nav>
    <a
      className="navLink"
      href="#/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="navLink"
      href="#/Calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className="navLink"
      href="#/Quotes"
      onClick={[Function]}
    >
      Quote
    </a>
  </nav>
</div>
`;
